backends-velox/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <artifactId>gluten-parent</artifactId>
     <groupId>io.glutenproject</groupId>
-    <version>0.5.0-SNAPSHOT</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
can we use a maven property to centralize version changes in the future?

Reply:
Thanks for your comment! It looks some open source projects (spark, hadoop, arrow-java) keep such explicitly setting. Maybe, there are some unknown considerations. Just keep such setting?